src/Transloadit.js

```diff
@@ -216,7 +216,7 @@ class TransloaditClient {
         checkResult(result)
 
         if (isResumable && Object.keys(allStreamsMap).length > 0) {
-          await sendTusRequest({
+          await tus.sendTusRequest({
```

Comment:
why is this needed?

Reply:
Module mocking works differently. In the follow-up PR that converts the project to TypeScript, will enable [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax) (for correctness). That means destructuring `require()` is no longer possible anyway.